e2e/linodeobjectstoragebucket-controller/minimal-linodeobjectstoragebucket/chainsaw-test.yaml

```diff
@@ -0,0 +1,56 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/kyverno/chainsaw/main/.schemas/json/test-chainsaw-v1alpha1.json
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  creationTimestamp: null
+  name: minimal-linodeobjectstoragebucket
+spec:
+  template: true
+  steps:
+  - name: step-00
+    try:
+    - assert:
+        file: 00-assert.yaml
+  - name: step-01
+    try:
+    - apply:
+        file: 01-create-linodeobjectstoragebucket.yaml
+    - assert:
+        file: 01-assert.yaml
+  - name: step-02
+    try:
+    - script:
+        env:
+        - name: TARGET_API
+          value: api.linode.com
+        - name: TARGET_API_VERSION
+          value: v4beta
+        - name: NAMESPACE
+          value: ($namespace)
```

Comment:
Question: Where are we passing in the namespace value from?

Reply:
This is a built-in variable chainsaw

Reply:
yes, `$namespace` is provided by chainsaw